Source/Orts.Simulation/Common/Events.cs

```diff
@@ -352,6 +340,9 @@ public static Event From(bool mstsBinEnabled, Source source, int eventID)
                         case 62: return Event.UncoupleB;
                         case 63: return Event.UncoupleC;
                         // Event 64 is unused in MSTS.
+                        // MSTSBin codes 66, 67 (documented at http://mstsbin.uktrainsim.com/)
+                        case 66: return Event.Pantograph2Up;
+                        case 67: return Event.Pantograph2Down;                        // Event 25 is possibly a vigilance reset in MSTS sound files but is never used.
```

Comment:
Looks like the event 25 comment from earlier has ended up on the end of this line by mistake

Reply:
Thanks.
Fixed in next commit